src/components/RadioBlock/RadioBlock.jsx

```diff
@@ -70,32 +103,62 @@ class RadioBlockItem extends Component {
 	}
 
 	onClick () {
+		if (this.props.disabled) {
+			return;
+		}
+
 		this.props.onClick(this.props.value);
 	}
 
+	renderOptionalTooltipAndContent (content) {
+		if(this.props.tooltipContent) {
+			return (
+				<FlyTooltip
+					content={this.props.tooltipContent}
+					position="top"
+					hoverIntent={true}
+					className={styles.RadioBlock_Option_TooltipWrapper}
+					widthIsAuto={true}
+				>
+					{ content }
```

Comment:
Tiny styling issue here. We usually don't use spaces inside JSX curly braces.

Reply:
Yes, good catch! Will fix.